CSharpSourceCode/Utilities/Extensions/SettlementExtensions.cs

```diff
@@ -33,5 +33,19 @@ public static List<string> GetSceneNames(this Settlement settlement)
             }
             return sceneNames;
         }
+
+        public static bool IsEmpireSettlement(this Settlement settlement)
+        {
+            return (settlement.IsTown || settlement.IsVillage) &&
+                   (settlement.MapFaction.Name.Contains("Moot") ||
+                    settlement.MapFaction.Name.Contains("Averland") ||
+                    settlement.MapFaction.Name.Contains("Stirland"));
+        }
```

Comment:
I suggest using the Culture instead of Faction Name. You can access Culture directly from settlement through a property.

Reply:
agree

Reply:
Settlement doesn't change its culture when it's conquered, so just imagine empire settlement with vampire culture. Currently my solution is pretty acceptable i think, but then we'll rework it for sure.